design-library/source/_patterns/01-atoms/04-blog-tag/blog-tag.twig

```diff
@@ -1 +1,6 @@
-<span class="highlight-title">{{ tag_name }}</span>
\ No newline at end of file
+<a class="highlight-title" style="background-color: {{ tag_background }}" href="{{ tag_link_url }}">
```

Comment:
I generally avoid using "style" in an element. We could use a class name that represents the possible tag backgrounds, e.g. `tag-background-blue` or `tag-background-green`. This could go into the "base" scss, e.g. https://github.com/nerdstein/nerdstein-design-library/blob/master/design-library/source/scss/base/_variables.scss